code/datums/spells/summon_supermatter.dm

```diff
@@ -0,0 +1,15 @@
+/obj/effect/proc_holder/spell/aoe/conjure/summon_supermatter
+	name = "Summon Supermatter Crystal"
+	desc = "Summons an active supermatter crystal. Imbues you with some supermatter, giving you resistance to it's hallucinations and radiation immunity."
+	base_cooldown = 5 MINUTES
+	cooldown_min = 60 SECONDS
+	summon_type = list(/obj/machinery/atmospherics/supermatter_crystal/shard)
+	summon_amt = 1
+	aoe_range = 0
+
+/obj/effect/proc_holder/spell/aoe/conjure/summon_supermatter/cast(list/targets, mob/living/user = usr)
+	..()
+	for(var/turf/our_target_turf in targets)
+		for(var/obj/machinery/atmospherics/supermatter_crystal/our_crystal in our_target_turf.contents)
```

Comment:
what if the crystal moves immediately after being summoned?

Reply:
I can't really think of a way this would happen bc this happens in the same proc after it's summoned, i just didn't wanna snowflake it too hard so it checks turf

Reply:
can you do me a solid and do a slight slight change to the base proc behaviour for conjure casts. Make the parent proc return a list of the summoned objects, that way you can just iterate through that. This way we can make sure we're avoiding hard refs but also potentially doing a very sus search and find which could potentially be abused if you summon two crystals on each other. (you could supercharge a crystal up another 3000 EER by summoning this ontop of another crystal)

Reply:
yeah sure

Reply:
done